posts/2020/11/Design In Open Source

```diff
@@ -0,0 +1,187 @@
+ <!--
+.. title: Introduction to Design in Open-Source
+.. slug: introduction-to-design-in-open-source
+.. date: 2020-11-10 00:07:30 UTC-05:00
+.. author: Tim George and Isabela Presedo-Floyd
+.. tags: design,UX, User Experience, Open-Source, 
+.. category: 
+.. link:
+.. description:
+.. type: text
+-->
+## Designers should be a good fit for open source
+*This blog post is a conversation. Portions lead by Tim George are marked with **TG**, and those 
+lead by Isabela Presedo-Floyd are marked with **IPF**.*
+
+**TG:** When I speak with other designers, one common theme I see concerning why they chose this 
+career path is they want to make a difference in the world. We design because we imagine a better 
+world and we want to help make it real. Part of the reason we design as a career is we’re 
+unable to go through life without designing; we’re always thinking about how things are and how 
+they could be better. This ethos also exists in many open-source communities. It seems like it ought 
+to be an ideal match.
+
+So what’s the disconnect? I'm still exploring that myself, but after a few years in open-source I want 
+to share my observations, experiences, and hope for a stronger collaboration between design and 
+development. I don’t think I have a complete solution, and some days I'm not even sure I grasp the 
+entire problem. What I hope is to say that which often goes unsaid in these spaces: design and 
+development skills in open source coexist precariously. 
+
+## Why design might be missing in open source
+**TG:** If you’re looking for a quick and singular answer, it might be best summed up by a quote from 
+Marshal McLuhan in his book The Medium is the Message: “We shape our tools, then our tools shape us.”  
+Github has become the place many open source projects exist. Github (I don’t want to suggest that  
+Github is the only place open-source exists, just that it’s the only place I’ve been involved with it)  
+is also designed first and foremost with code contributions in mind. The ripple effect is that the  
+people and the project will similarly center around code and it will take effort to use the tool in  
+a way it is not designed to support. While it has a number of features that support design input,  
+it doesn’t integrate directly with design tools, and it isn’t tracked in the git history in any meaningful  
+way. Unless, of course, the designer is directly committing code, but then the design process still  
+isn’t accurately reflected, only the outcome of it.
+
+**IPF:** That may be one theory as to the problem’s root, but its certainly not the only explanation.  
+Because the question of missing skills in open source could be the source of many blog posts, I’ve  
+narrowed it down to a list. Topics marked with an * will be expanded upon later. Here are some other  
```

Comment:
Since you are presenting the list you just referred to, I wouldn't say they are "other" thoughts (makes me think like I missed the first points you were about to make).

nit: Is there a reason to order the items this way? Maybe the points to be expanded on later can be grouped together somehow?

Reply:
The ordering is not currently opinionated. I like your idea, let me work some on that.